_config.yml

```diff
@@ -1 +1,3 @@
 baseurl: ""
+
+theme: jekyll-theme-cayman
```

Comment:
Do not modify this file. remove this line.

Reply:
which line i've to remove

Reply:
as a whole

Reply:
the `theme: jekyll-theme-cayman` line

Reply:
and squash your commits.

Reply:
ok

Reply:
done